extensions/modules/expathrepo/src/test/java/org/exist/repo/PackageTriggerTest.java

```diff
@@ -50,7 +50,7 @@
 
 public class PackageTriggerTest {
 
-    static final String xarFile = "triggertest-1.1.0.xar";
+    static final String xarFile = "exist-expathrepo-trigger-test-5.4.0-SNAPSHOT.xar";
```

Comment:
version is hardcoded... can we derive this from the environment ?

Reply:
@dizzzz Ah yes, good point - I forgot that I left that in... let me fix...

Reply:
Okay I sent a follow-up commit to fix that.